tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.Format.ps1xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Configuration>
+  <ViewDefinitions>
+    <View>
+      <Name>ProfileMapDataView</Name>
+      <ViewSelectedBy>
+        <TypeName>ProfileMapData</TypeName>
+      </ViewSelectedBy>
+      <ListControl>
+        <ListEntries>
+          <ListEntry>
+            <ListItems>
+              <ListItem>
+                <PropertyName>ProfileName</PropertyName>
+              </ListItem>
+              <ListItem>
+                <Label>Modules</Label>
+                <ScriptBlock> 
+                  $RequiredModules = "AzureRM", "AzureRM.Storage", "AzureRM.Compute", "AzureRM.Network"
```

Comment:
A ListItem inside the format.ps1xml file only displays 4 items from the scriptblock. Because of this limitation, we wouldn't be able to add more than 4 module names